src/frontend/apps/desk/src/auth/useAuthStore.tsx

```diff
@@ -27,6 +28,10 @@ const useAuthStore = create<AuthStore>((set) => ({
 
       return {};
     }),
+
+  logout: () => {
+    set({ authenticated: false, token: null, initialized: false });
```

Comment:
could be factorized with the default state used when creating the store

Reply:
Done:
https://github.com/numerique-gouv/people/blob/f77630cc4f3630daaf85ada897c0092eecb81776/src/frontend/apps/desk/src/auth/useAuthStore.tsx